app/components/image-page.js

```diff
@@ -1,5 +1,5 @@
 import Component from '@ember/component';
 
 export default Component.extend({
-  classNames: [ 'image-page' ]
+  classNames: [ 'image-page-community' , 'image-page-thankyou', 'image-page-faq']
```

Comment:
To create multiple background images, I edited the classNames so that we can add different bkg image to Community, Thankyou, and FAQ page